src/lib/jsonParse/jsonParse.cpp

```diff
@@ -378,7 +378,7 @@ static std::string jsonParse
 
```

Comment:
Please included an entry in CHANGES_NEXT_RELEASE file regarding the fix. For instance:

```
- Fix: bug that may contextBroker to crash when passing inappropriate parameter in query context payload (#3055)
```

The content of that file is eventually used to generate changelog between versions so it is good to track changes/issues fixed or implemented there.

Reply:
@fgalan I've done that. Thanks for your valuable comment.
Please let me know if any further changes are required.

Reply:
According to https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/contribution_guidelines.md#pull-request-protocol you should end this comment thread with a `"Fixed in <commit_hash>"` comment.

> * After discussion, the comment thread ends in one of the following ways:
>    * `Fixed in <commit hash>` in case the discussion involves a fix in the PR branch (which commit hash is included as reference)
>    *  `NTC`, if finally nothing needs to be done (NTC = Nothing To Change)

(It is a very minor thing but it worth it to get familiarity with the contribution process for future PRs ;)

Reply:
Fixed in 767771823ba4f93291b77da32b72e1fa8764ab0b